uwblueprint/social-diversity-for-children
# feat(admin view registrants): page + hooks

### Notion ticket link



[View/browse list of registrants](https://www.notion.so/uwblueprintexecs/View-browse-list-of-registrants-72eefeb3c228474c8d79ccfcca4f1c6e)



### Implementation description

https://user-images.githubusercontent.com/44826218/140867761-7b8c808b-1915-4445-a809-6d9d2b9abea8.mp4

- Reused react table for browse registrant page
- Add getUsers hook for admin usage
- new page under /admin/registrant
- fix wrapper to be min 100vw by default
- Add new table data hooks for parent, volunteer and student data



### Steps to test

1. Populate app with parents, guardians, students
2. Make sure that users/students shows up in their corresponding tab. All links link to a user profile (parent/volunteer) or links to parent profile (for student)
3. Test export by pressing the export button, a csv file should be downloaded, showing all the data in the table
4. Global filter input should be working as expected



### What should reviewers focus on?

- Dev: Table Reuse
- Design: Look and Feel of browse registrants

### Checklist
